ktor-server/ktor-server-netty/jvm/test/io/ktor/tests/server/netty/NettyHttp3MultipleConnectionsTest.kt

```
@@ -0,0 +1,170 @@
+                .handler(codec)
+                .bind(0).sync().channel()
+            val quic: QuicChannel
+            val ms = measureTimeMillis {
+                quic = QuicChannel.newBootstrap(udp)
+                    .handler(Http3ClientConnectionHandler())
+                    .remoteAddress(InetSocketAddress("127.0.0.1", sslPort))
+                    .connect().get(5, TimeUnit.SECONDS)
+            }
+            println("[probe] $label: connected in ${ms}ms")
+            return H3Conn(group, udp, quic)
+        } catch (cause: TimeoutException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)